replication/binlogsyncer.go

```diff
@@ -57,6 +57,18 @@ type BinlogSyncerConfig struct {
 	// We will use Local location for timestamp and UTC location for datatime.
 	ParseTime bool
 
+	// If ParseTime is false, convert the datetime/timestamps into this specified
```

Comment:
do we need to pass this to the fracTime no matter ParseTime true or false?

Reply:
No we don't. that's why I only pass it down when parseTime == false